xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/store/XWikiStoreInterface.java

```diff
@@ -400,6 +400,11 @@ List<XWikiDocument> searchDocuments(String wheresql, boolean distinctbylanguage,
 
     void deleteLock(XWikiLock lock, XWikiContext context, boolean bTransaction) throws XWikiException;
 
+    /**
+     * @deprecated links indexing move to Solr, see org.xwiki.refactoring.internal.solr.* in
```

Comment:
Shouldn't you point to the new xwiki-platform-link module now instead?

Reply:
For the load*links yes. The message will me a bit different for the write APIs (since there is no write API anymore).